### PR TITLE
Add optimade-client dependency.

### PR DESCRIPTION
**Pull request by yakutovicha**
This PR should fix the Travis tests.

### CODE REVIEW

File: setup.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "more_itertools",
         "nglview",
         "numpy",
+	"optimade-client",
```
Comment (csadorf) [inline setup.json:22]: Looks like the indentation is wrong.

Comment (yakutovicha) [reply setup.json]: Indeed, that was a Tab. Fixed.

**Review by csadorf — APPROVED** [review]
Looks good. However, I'm wondering whether overall we should be provide ranges here. All of these dependencies are very vague without version numbers.

**Comment by yakutovicha** [comment]
> Looks good. However, I'm wondering whether overall we should be provide ranges here. All of these dependencies are very vague without version numbers.

I agree, made an issue to not forget about this: https://github.com/aiidalab/aiidalab-widgets-base/issues/146

**Pull request merged**